equinoxfitness/datacoco-db
# Datplat 1292

## Changes

removed cache_manager, redis_tools, sf_tools
Added tests from our production repo
updated readme to reflect changes
remove datacoco-core dependency

### How Has This Been Tested?

1. Checkout this branch
2. Switch to virtual environment
3. Run `pip install -r requirements-dev.txt`
 - run tox and verify all tests run
